ssv/queue/message_prioritizer.go

```diff
@@ -0,0 +1,193 @@
+package queue
+
+import (
```

Comment:
should that be in ssv or outside as a separate module?

Reply:
Not sure, I guess because it's sorting both `qbft.SignedMessage` and `ssv.PartialSignatureMessage` it should be at top level?